utils/cliutils/cli_consts.go

```diff
@@ -19,4 +19,7 @@ const (
 
 	// Common
 	Retries = 3
+
+	// Env
+	ReportUsage = "JFROG_CLI_REPORT_USAGE"
```

Comment:
It is really great that you placed this new env var here!
Why don't we take this opportunity and move all env vars here?

Reply:
I've moved the JFROG_CLI_LOG_LEVEL, JFROG_CLI_OFFER_CONFIG, JFROG_CLI_HOME_DIR and JFROG_CLI_HOME as well